### DISCUSSION TITLE
How to specific schema files to be reused by other projects.

### DISCUSSION THREAD

**Discussion by baucosta** [99designs/gqlgen#1803] [Q&A]
Hi there!!!

So, I wanna load schema files in a different project. I'm developing two APIs which both will use the same schema files. How can I load remote schema files in more of one project?

I've tried it in gqlgen.yml:

schema:
  - github.com/group/project/graphql/schemas/*.graphql

However, it did not work.

Thanks a lot.

**Comment by HuanLiu-hotstar** [comment]
I have the same issue
